components/swatch/stories/swatch.stories.js

```diff
@@ -49,23 +57,193 @@ export default {
 			control: "boolean",
 		},
 		isSelected,
+		borderStyle: {
+			name: "Border style",
+			type: { name: "string" },
+			table: {
+				type: { summary: "string", required: true },
+				category: "Component",
+			},
+			options: ["default", "noBorder", "lightBorder"],
+			control: "select",
+		},
+		isRectangle: {
+			name: "Rectangle",
+			type: { name: "boolean" },
+			table: {
+				type: { summary: "boolean" },
+				category: "Component",
+			},
+			control: "boolean",
+		},
+		imageUrl: {
+			name: "Image url",
+			type: { name: "string" },
+			table: {
+				type: { summary: "string" },
+				category: "Content",
+			},
+			control: { type: "file", accept: ".svg,.png,.jpg,.jpeg,.webc" },
+		},
+		gradient: {
+			name: "Gradient values",
+			description: "Input an example gradient.",
+			type: { name: "string" },
+			table: {
+				type: { summary: "string" },
+				category: "Component",
+			},
+			control: "color",
+		},
+		isMixedValue: {
+			name: "Mixed value",
+			type: { name: "boolean" },
+			table: {
+				type: { summary: "boolean" },
+				category: "Component",
+			},
+			control: "boolean",
+		},
```

Comment:
Sure. There's certainly not much of anything for mixed value, so let me know if this is ok. I also added some descriptions for corner rounding and the imageUrl here: https://github.com/adobe/spectrum-css/pull/2925/commits/0ab2a7e463ba6b03da403fbb8e980608d0fde262

Again, there's not much information for the rectangle. I decided maybe it would be clearer to change the name of that control from "Rectangle" to "Shape." Those changes are here, along with the description update: https://github.com/adobe/spectrum-css/pull/2925/commits/4aef94f5aa3409bf3dbd5580ad5faf851dbfd819

I also did a little refactoring on the gradient (since there was another question on that control anyways). What do you think of the updates? There's an updated description, and it's a text control now (which is probably clearer that you can type into it). https://github.com/adobe/spectrum-css/pull/2925/commits/c4ecea08c46c4c9b8f99aa2a24cd9ff5db7c3dc3